lib/SGN/Controller/AJAX/BrAPI.pm

```diff
@@ -589,9 +589,23 @@ sub observation_levels_GET {
 sub seasons : Chained('brapi') PathPart('seasons') Args(0) : ActionClass('REST') { }
 
 sub seasons_POST {
-    my $self = shift;
-    my $c = shift;
-    seasons_process($self, $c);
+	my $self = shift;
+	my $c = shift;
+	my $clean_inputs = $c->stash->{clean_inputs};
+	my $postedData = $clean_inputs;
+	my @data;
+	foreach my $season (values %{$postedData}) {
+		push @data, {
+			seasonDbId=>$season->{year},
+			season=>undef,
```

Comment:
I take it we don't care about the seasonName being returned?

Reply:
Looking at the save functionality, it looks like only the season year is saved for a given study, so that's why I hadn't included season in the response on the POST endpoint

Reply:
sounds good